python/ray/tests/test_placement_group.py

```diff
@@ -418,5 +425,51 @@ def value(self):
     ray.shutdown()
 
 
+def test_check_bundle_index(ray_start_cluster):
+    @ray.remote(num_cpus=2)
+    class Actor(object):
+        def __init__(self):
+            self.n = 0
+
+        def value(self):
+            return self.n
+
+    cluster = ray_start_cluster
+    num_nodes = 2
```

Comment:
I don't think we need multiple nodes for this. Let's just do it in a single node (so that we can shorten testing time).

Reply:
fixed